packages/viewer/src/types/frame.ts

```diff
@@ -29,15 +30,19 @@ export interface SceneAttributes {
 }
 
 export const fromProto = (
-  payload: vertexvis.protobuf.stream.IDrawFramePayload
+  payload: vertexvis.protobuf.stream.IDrawFramePayload,
+  isDepthBuffer = false
```

Comment:
Why is this a flag vs just including it if the depth buffer is set?

Reply:
Yeah, probably doesn't need to be a flag. Was using this due to the canvas renderer expecting `image` to be populated with the data to draw, but I'll update to do that when passing the depth buffer instead